data/ignore_ids_safety_scan.json

```diff
@@ -606,7 +606,8 @@
                 "44715":"for shipping numpy 1.21.5 - the last available version for py3.7",
                 "44716":"for shipping numpy 1.21.5 - the last available version for py3.7",
                 "44717":"for shipping numpy 1.21.5 - the last available version for py3.7",
-                "43453":"[DISPUTED] The vendor does not agree this is a vulneraility; In (very limited) circumstances a user may be able provoke the buffer overflow, the user is most likely already privileged to at least provoke denial of service by exhausting memory. Triggering this further requires the use of uncommon API (complicated structured dtypes), which is very unlikely to be available to an unprivileged user."
+                "43453":"[DISPUTED] The vendor does not agree this is a vulneraility; In (very limited) circumstances a user may be able provoke the buffer overflow, the user is most likely already privileged to at least provoke denial of service by exhausting memory. Triggering this further requires the use of uncommon API (complicated structured dtypes), which is very unlikely to be available to an unprivileged user.",
+                "48551": "tensorflow-estimator and tensorflow versions must match. For all TF versions below TF 2.9.0, we cannot upgrade tf-estimator to 2.9.0"
```

Comment:
Is this needed for the inference images? Do the inference images have tf estimator?

Reply:
I saw this for the build image.
`ERROR    test.dlc_tests.sanity.test_safety_report_file:test_safety_report_file.py:94 SAFETY_REPORT (FAILED) [pkg: tensorflow-estimator] [installed: 2.6.0] [vulnerabilities: [SafetyVulnerabilityAdvisory(vulnerability_id='48551', advisory='Tensorflow-estimator 2.9.0 removes all boosted trees code due to security issues.', reason_to_ignore='N/A', spec='<2.9.0')]]`
should Tensorflow-estimator only exists in training image?

Reply:
it's the dependency of the TF.
@philschmid is the TF wheel installed a must in HF TF inference images? for other inference images, only TFS is needed

Reply:
FYI: TF wheel been installed from HF TF2.4

Reply:
huggingface use MMS + Framework + HF stuff to do inference instead of TFS